pkg/cache/context.go

```diff
@@ -706,6 +706,15 @@ func (ctx *Context) updateApplicationTags(request *interfaces.AddApplicationRequ
 			request.Metadata.Tags[siCommon.AppTagNamespaceResourceQuota] = string(quotaStr)
 		}
 	}
+
+	// add guaranteed resource info as an app tag
+	guaranteedResource := utils.GetNamespaceGuaranteedFromAnnotation(namespaceObj)
+	if guaranteedResource != nil && !common.IsZero(guaranteedResource) {
+		if guaranteedStr, err := json.Marshal(guaranteedResource); err == nil {
```

Comment:
Nit: `guaranteedStr` is not a string, it's a byte array. Just call it `guaranteed`.